Tests/AnimationValueSpec.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  AnimationValueSpec.swift
+//  Lotus
+//
+//  Created by Vladislav Kondrashkov on 2/14/20.
+//  Copyright © 2020 Vladislav Kondrashkov. All rights reserved.
+//
+
+import Quick
+import Nimble
+@testable import Lotus
+
+final class AnimationValueSpec: QuickSpec {
+    override func spec() {
+        describe("on animationValueFor(:)") {
+            var animationValue: AnimationValue!
+
+            beforeEach {
+                animationValue = 0
+            }
+
+            it("should return float value when int given") {
```

Comment:
Lets replase "int" "unsigned int" and so on with the actuall type "Int", "UInt").  Easier to read this way.